src/mame/drivers/vgmplay.cpp

```diff
@@ -1159,10 +1172,15 @@ void vgmplay_state::machine_start()
 			m_ym2612->set_unscaled_clock(r32(0x2c));
 		if(version >= 0x110 && r32(0x30))
 			m_ym2151->set_unscaled_clock(r32(0x30));
-
+			
 		if(version >= 0x151 && r32(0x38))
 			m_segapcm->set_unscaled_clock(r32(0x38));
 
+		if(version >= 0x151 && r32(0x3c)) {
```

Comment:
This doesn't match the style of the surrounding code.  The statement immediately below is Allman style.